tests/unit_tests/test_model.py

```diff
@@ -185,8 +185,8 @@ def test_get_all_models(mock_http_client):  # type: (mock.Mock) -> None
       "app_id": "main",
       "output_info": {
         "message": "Show output_info with: GET /models/{model_id}/output_info",
-        "type": "facedetect",
-        "type_ext": "facedetect"
+        "type": "detect",
+        "type_ext": "detect"
```

Comment:
Do a `detect` type & type_ext really exist? I only see `detection`.

In any case, it's a unit test so it's not very important what the exact value is.

Reply:
true, yeah they will :) going to clean that up too eventually in the API. Not sure why we return `detection` but then `detect-embed` and `detect-concept`.